kumascript/macros/CSSRef.ejs

```diff
@@ -1264,6 +1268,16 @@ async function buildPropertylist(pages, title) {
           </ol>
       </details>
   </li>
+  <li class="toggle">
+    <details>
+        <summary><%=text['Containment']%></summary>
+        <ol>
+          <li><%-smartLink(`${cssURL}CSS_Containment/Using_CSS_containment`, null, text['Using_CSS_containment'], cssURL)%></li>
+          <li><%-smartLink(`${cssURL}CSS_Containment/Container_queries`, null, text['Container_queries'], cssURL)%></li>
+          <li><%-smartLink(`${cssURL}CSS_Containment/Container_size_and_style_queries`, null, text['Container_size_and_style_queries'], cssURL)%></li>
```

Comment:
This should work and automatically use the `title` (or `short-title`) of the page, can you try? (Then we don't need the titles above.)

```suggestion
          <li><%-smartLink(`${cssURL}CSS_Containment/Using_CSS_containment`)%></li>
          <li><%-smartLink(`${cssURL}CSS_Containment/Container_queries`)%></li>
          <li><%-smartLink(`${cssURL}CSS_Containment/Container_size_and_style_queries`)%></li>
```